Workbooks/ThycoticWorkbook.json

```diff
@@ -0,0 +1,322 @@
+            "formatter": 1,
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          },
+          "nodeIdField": "countRecord",
+          "sourceIdField": "Activity",
+          "targetIdField": "FileType",
+          "graphOrientation": 3,
+          "showOrientationToggles": false,
+          "nodeSize": null,
+          "staticNodeSize": 100,
+          "colorSettings": null,
+          "hivesMargin": 5
+        }
+      },
+      "name": "query - 3"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Expires secrets"
+      },
+      "name": "text - 5"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where LogSeverity == 2 \r\n| where TimeGenerated > ago(1d)\r\n| summarize count() by Activity\r\n| where extract(\"EXPIRE[S|D](\\\\d+)DAY\\\\w?\", 1, Activity) != \"\"\r\n| project extract(\"EXPIRE[S|D](\\\\d+)DAY\\\\w?\", 1, Activity), count_\r\n| order by count_ asc ",
+        "size": 2,
+        "noDataMessage": "Secrets that will soon expire are not found",
+        "noDataMessageStyle": 3,
+        "timeContext": {
+          "durationMs": 86400000
+        },
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "piechart",
+        "chartSettings": {
+          "seriesLabelSettings": [
+            {
+              "seriesName": "03",
+              "label": "Expre to 3 days",
+              "comment": "Expire to 3 days"
+            },
+            {
+              "seriesName": "07",
+              "label": "Expire to 7 days",
+              "comment": "Expire to 7 days"
+            },
+            {
+              "seriesName": "15",
+              "label": "Expire to 15 days",
+              "comment": "Expire to 15 days"
+            },
+            {
+              "seriesName": "30",
+              "label": "Expire to 30 days"
+            },
+            {
+              "seriesName": "01",
+              "label": "Expire to 1 day"
+            }
+          ],
+          "ySettings": {
+            "numberFormatSettings": {
+              "unit": 17,
+              "options": {
+                "style": "decimal",
+                "useGrouping": true
+              }
+            }
+          }
+        }
+      },
+      "name": "query - 4"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "### Expired secret today"
+      },
+      "name": "text - 7"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "CommonSecurityLog\r\n| where LogSeverity == 2 \r\n| where TimeGenerated > ago(1d)\r\n| summarize count() by Activity\r\n| where Activity == \"SECRET - EXPIREDTODAY\"\r\n| project count_\r\n| order by count_ asc ",
```

Comment:
Same comment as the sample query, please make sure the query only targets logs from Thycotic